infrastructure/merklemountainrange/tests/unit/mmr.rs

```diff
@@ -41,9 +41,120 @@ fn create_small_mmr() {
     let hash0 = mmr.get_hash(0).unwrap();
     let proof = mmr.get_hash_proof(&hash0);
     let mut our_proof = Vec::new();
-    our_proof.push(mmr.get_hash(0).unwrap());
-    our_proof.push(mmr.get_hash(1).unwrap());
-    our_proof.push(mmr.get_hash(2).unwrap());
-    assert_eq!(hash_values.get_slice(2), HashValues::to_hex_multiple(&proof));
+    for i in 0..3 {
+        our_proof.push(mmr.get_hash(i).unwrap());
+    }
+    assert_eq!(hash_values.get_slice(0, 2), HashValues::to_hex_multiple(&proof));
     assert_eq!(mmr.verify_proof(&our_proof), true);
 }
+
+#[test]
+fn create_med_mmr() {
```

Comment:
what is med?

Reply:
medium

Reply:
> Looks fine, I cannot really confirm the indexing logic in this review but the tests seem like they will confirm it.
> 
> Mainly I think the support stuff that is used in the tests should be in a test module with the #[cfg(test)] flag.

All the tests are integration tests, which means they are in module/test folder
This means that rust doesnt require the #[cfg(test)]  to know its test